src/Observer/AccountableObserver.php

```diff
@@ -12,12 +13,19 @@ class AccountableObserver
      */
     protected $config;
 
+    /**
+     * @var AccountableStatus
+     */
+    protected $status;
+
     /**
      * AccountableObserver constructor.
      */
     public function __construct()
     {
         $this->config = config('accountable');
+
+        $this->status = app()->make(AccountableStatus::class);
```

Comment:
Deze status abstraheert de config repo, maar de config zelf wordt wel gebruikt voor de kolomnamen?

Reply:
Klopt, beetje gepikt van spatie. Ik heb van dit ding een singleton gemaakt en er een helpertje bij gezet. Nu kan je snel zeggen: ```accountable()->disable(); ```

Reply:
Het heeft ook als doel om config te abstraheren, denk ik. Of niet?